hybridauth/Hybrid/Auth.php

```diff
@@ -352,6 +352,9 @@ public static function logoutAllProviders() {
 	 * @param string $mode PHP|JS
 	 */
 	public static function redirect($url, $mode = "PHP") {
+		if(!$mode){
+			$mode = 'PHP';	

```

Comment:
Exactly, in some cases **$mode** is passed as an empty string, and in this case **$mode** will not be initialized by '_PHP_' value since it is not **null**